content/en/docs/setup/_index.md

```diff
@@ -48,6 +48,7 @@ If you're learning Kubernetes, use the Docker-based solutions: tools supported b
 |                     | [IBM Cloud Private-CE (Community Edition) on Linux Containers](https://github.com/HSBawa/icp-ce-on-linux-containers)|
 |                     | [k3s](https://k3s.io)|
 |                     | [Ubuntu on LXD](/docs/getting-started-guides/ubuntu/)|
+|                     | [kind k8s IN Docker](https://github.com/kubernetes-sigs/kind)|
```

Comment:
Couple of comments.
I suggest hyperlinking to https://kind.sigs.k8s.io/
Also, I'd make the link text be “kind” or “KinD” and follow it with “(Kubernetes in Docker)”.

What do you think @medmedchiheb ?

Reply:
Hold on, I just noticed that Kubernetes IN Docker is already there in the other column.

Reply:
ok then, let's revert and change the first one "link and name". i agree with you it's better